src/main/java/com/nike/cerberus/client/CerberusClient.java

```diff
@@ -227,7 +255,12 @@ public CerberusResponse read(final String path) {
         final HttpUrl httpUrl = buildUrl(SECRET_PATH_PREFIX, path);
         logger.debug("read: requestUrl={}", httpUrl);
 
-        final Response response = executeWithRetry(httpUrl, HttpMethod.GET, null, DEFAULT_NUM_RETRIES, DEFAULT_RETRY_INTERVAL_IN_MILLIS);
```

Comment:
Do we still need this function? it didn't get deleted

Reply:
good catch. Fixed.